tests-e2e/ocw-ci-test-course/shortcodes.spec.ts

```diff
@@ -37,3 +37,68 @@ test("Related resources link to correct page", async ({ page }) => {
   const href = await resourceLink.getAttribute("href")
   expect(href).toBe("/courses/ocw-ci-test-course/resources/example_pdf/")
 })
+
+test("Start and end times does not exist", async ({ page }) => {
```

Comment:
**Requested Change:** Could you move these tests to `video.spec.ts`? They aren't really shortocde tests.